### PR TITLE
help-azure update for juju rc1

### LINKED ISSUE #1375: Azure help pages will need to change - userpass deprecated
Currently the guide for configuring Azure uses username/password, but this will no longer be supported.

### PR DESCRIPTION
**Pull request by degville**
fixes #1375 

- added/tested instructions for new azure interactive authentication in rc1
- changed install priority and updated userpass and service-principal-secret details.
- replaced/updated images

### CODE REVIEW

File: src/en/help-azure.md [hunk 1]
```diff
@@ -26,7 +26,111 @@ juju update-clouds
 
 ## Credentials
 
-In order to access Azure, you will need to add some credentials for Juju to use.
+Using Juju's interactive authentication, importing Azure credentials into Juju
+is now a simple process. The only information you'll need is your Azure subscription
```
Comment (axw) [inline src/en/help-azure.md]: Is the "now" relevant to people who haven't used the provider before? That seems more like a comment for release notes.

File: src/en/help-azure.md [hunk 2]
```diff
@@ -26,7 +26,111 @@ juju update-clouds
...
+either type 'interactive' manually, or simply press 'Enter' to continue. 
+
+!!! Note: The 'interactive' option is far quicker and easier than manually adding
+credentials via either the 'service-principal-secret' or 'userpass'
+options, but instructions for these are covered in the 
+[Manually adding credentials](#manually-adding-credentials) section.
+
+You will then be asked for your Azure subscription id.  After entering this, you'll
+be notified that Juju is initiating its interactive authentication followed by
+a request to use a web browser to follow a [link][azuredeviceauth] and enter an
```
Comment (axw) [inline src/en/help-azure.md]: s/to follow a/to follow/ ?

File: src/en/help-azure.md [hunk 3]
```diff
@@ -26,7 +26,111 @@ juju update-clouds
...
+ - application-id
+ - subscription-id
+ - application-password
+
+Additionally, if you selected `userpass', you will also need:
+
+ - tenant-id
+
+!!! Note: The 'userpass' authentication type is being deprecated and will be
+remove soon.
```
Comment (axw) [inline src/en/help-azure.md]: removed

**Comment by degville** [comment]
Updated as per comments from @axw  - thanks for the review.

File: src/en/help-azure.md (hunk 3 above)
Comment (matthewhelmke) [inline src/en/help-azure.md]: @degville s/remove/removed

**Comment by matthewhelmke** [comment]
LGTM
:beers:

**Pull request merged**